libs/auth/src/lib/services/auth.service.ts

```diff
@@ -44,6 +47,10 @@ export class AuthService {
           // If something else has interrupted the auth process, then we don't want to signal a success.
           if (this._state$.value === AuthState.Ongoing) {
             this._state$.next(AuthState.Success);
+            this.dialogService.open<DisclaimerDialogComponent>(DisclaimerDialogComponent, {
```

Comment:
The type parameter can be dropped here.
```suggestion
         this.dialogService.open(DisclaimerDialogComponent, {
```